moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_arm_data.h

```diff
@@ -84,7 +85,7 @@ struct JogArmShared
   Eigen::Isometry3d tf_moveit_to_cmd_frame;
   
   // The dimesions to control. In the command frame. [x, y, z, roll, pitch, yaw]
-  std::vector<bool> control_dimensions{ true, true, true, true, true, true };
+  std::array<std::atomic_bool, 6> control_dimensions;
```

Comment:
It took me awhile to figure out the C++ voodoo but you can initialize the array like this:

```
  std::atomic_bool drift_dimensions [6] =
    {ATOMIC_VAR_INIT(false), ATOMIC_VAR_INIT(false), ATOMIC_VAR_INIT(false), ATOMIC_VAR_INIT(false), ATOMIC_VAR_INIT(false), ATOMIC_VAR_INIT(false)};
```

^ my example is slightly different

Reply:
Oh yea that is great, thanks! Changed and committed